hs_core/tests/api/rest/test_create_resource.py

```diff
@@ -55,6 +55,48 @@ def test_post_resource_get_sysmeta(self):
             self.assertEqual(response['Content-Type'], 'application/zip')
             self.assertGreater(int(response['Content-Length']), 0)
 
+    def test_post_resource_get_sysmeta(self):
+        rtype = 'GenericResource'
```

Comment:
@alvacouch That makes sense. Then if the whole "DEPRECATED" test will be deleted later, perhaps we can just remove the "DEPRECATED" test all together while only testing the new url pattern. This will remove the duplicate code and also avoid testing the same functionalities twice. Since the new url is tested, I'd think if tests for new url pass, they would pass for the old corresponding url as well.